### PR TITLE
[otbn,dv] Copy, rather than reference, insn-specific coverage points

### PR DESCRIPTION
**Pull request by rswarbrick**
This document was trying avoid duplication, so it tried to say things
like "ADDI has the same addition-related coverage points as ADD". This
has confused one reviewer already, and probably makes things harder to
track. Do the copy-paste.

### CODE REVIEW

File: hw/ip/otbn/doc/dv/index.md [hunk 1]
```diff
@@ -414,7 +435,8 @@ This instruction uses the `bnaf` encoding schema, with covergroup `enc_bna_cg`.
 
 This instruction uses the `bnai` encoding schema, with covergroup `enc_bnai_cg`.
 
-Specific points as for `BN.ADD`.
+- Extremal values of shift for both directions where the shifted value is nonzero
```
Comment (matutem) [inline hw/ip/otbn/doc/dv/index.md:438]: Shift? This is ADDI. Perhaps a copy-paste error.

File: hw/ip/otbn/doc/dv/index.md [hunk 2]
```diff
@@ -439,34 +461,39 @@ This instruction uses the `bnaq` encoding schema, with covergroup `enc_bnaq_cg`.
 This instruction uses the `bnaq` encoding schema, with an extra field not present in `bn.mulqacc`.
 Encoding-level coverpoints are tracked in covergroup `enc_bnaqw_cg`.
 
-Specific points as for `BN.MULQACC`, plus the generic flag group cover points.
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
```
Comment (matutem) [inline hw/ip/otbn/doc/dv/index.md:465]: What do you mean by "See the accumulator overflow"? Also above and velow.

File: hw/ip/otbn/doc/dv/index.md [hunk 3]
```diff
@@ -439,34 +461,39 @@ This instruction uses the `bnaq` encoding schema, with covergroup `enc_bnaq_cg`.
...
 - Cross the generic flag updates with `wrd_hwsel`, since the flag changes are different in the two modes.
 
 #### BN.SUB
 
 This instruction uses the `bnaf` encoding schema, with covergroup `enc_bna_cg`.
 
-Specific points as for `BN.ADD`.
+- Extremal values of shift for both directions where the shifted value is nonzero
```
Comment (matutem) [inline hw/ip/otbn/doc/dv/index.md:480]: Mention of shift seems out of place.

**Review by matutem — COMMENTED** [review]
Rupert, thanks for the clarifications. I left some comments asking for some clarifications and pointing to some apparent copy-paste errors.

File: hw/ip/otbn/doc/dv/index.md (hunk 1 above)
Comment (rswarbrick) [reply hw/ip/otbn/doc/dv/index.md:438]: This is BN.ADDI, which has a shift operand. Read [the ISA doc](https://docs.opentitan.org/hw/ip/otbn/doc/isa/#bnaddi) for more information.

File: hw/ip/otbn/doc/dv/index.md (hunk 2 above)
Comment (rswarbrick) [reply hw/ip/otbn/doc/dv/index.md:465]: We expect to see a `mulqacc.*` instruction that causes the (unsigned 256-bit) accumulator register to overflow when it is incremented. Is there another short formulation that you think would be clearer to someone who understands the ISA?

File: hw/ip/otbn/doc/dv/index.md (hunk 3 above)
Comment (rswarbrick) [reply hw/ip/otbn/doc/dv/index.md:480]: As with `bn.add`, [`bn.sub`](https://docs.opentitan.org/hw/ip/otbn/doc/isa/#bnsub) has a shift that's applied to one of its arguments.

**Review by imphil — APPROVED** [review]
LGTM, thanks!

**Comment by rswarbrick** [comment]
Hi @matutem, I think I've addressed your questions and I'm going to merge now. Let me know if I missed something and I'll file a follow-up.

**Pull request merged**